src/test/java/uk/gov/hmcts/reform/sscs/utils/SscsCaseDataUtilsTest.java

```diff
@@ -0,0 +1,381 @@
+        //then
+        assertEquals("", SscsCaseDataUtils.getHearingType(sscsCaseData));
+    }
+
+    @Test
+    void shouldGetIssueCode() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        // when
+        Mockito.when(sscsCaseData.getIssueCode()).thenReturn("DD");
+        //then
+        assertEquals(1, SscsCaseDataUtils.getIssueCode(sscsCaseData).size());
+        assertEquals("DD", SscsCaseDataUtils.getIssueCode(sscsCaseData).stream().findFirst().orElseThrow());
+    }
+
+    @Test
+    void shouldGetEmptyListWhenIssueCodeNull() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        // when
+        Mockito.when(sscsCaseData.getIssueCode()).thenReturn(null);
+        //then
+        assertTrue(SscsCaseDataUtils.getIssueCode(sscsCaseData).isEmpty());
+    }
+
+    @Test
+    void shouldGetFacilitiesRequired() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        Appeal appeal = Mockito.mock(Appeal.class);
+        HearingOptions hearingOptions = Mockito.mock(HearingOptions.class);
+        // when
+        Mockito.when(hearingOptions.getArrangements()).thenReturn(List.of("signLanguageInterpreter",
+                "hearingLoop", "disabledAccess"));
+        Mockito.when(appeal.getHearingOptions()).thenReturn(hearingOptions);
+        Mockito.when(sscsCaseData.getAppeal()).thenReturn(appeal);
+        //then
+        List<String> facilitiesRequired = SscsCaseDataUtils.getFacilitiesRequired(sscsCaseData);
+        assertEquals(3, facilitiesRequired.size());
+        assertEquals("signLanguageInterpreter", facilitiesRequired.stream().findFirst().orElseThrow());
+        assertEquals("disabledAccess", facilitiesRequired.stream().skip(2).findFirst().orElseThrow());
+    }
+
+    @Test
+    void shouldGetEmptyFacilitiesRequiredWhenAppealNull() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        // when
+        Mockito.when(sscsCaseData.getAppeal()).thenReturn(null);
+        //then
+        assertTrue(SscsCaseDataUtils.getFacilitiesRequired(sscsCaseData).isEmpty());
+    }
+
+    @Test
+    void shouldGetCaseName() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        Appeal appeal = Mockito.mock(Appeal.class);
+        Appellant appellant = Mockito.mock(Appellant.class);
+        Name name = Mockito.mock(Name.class);
+        // when
+        Mockito.when(name.getFullName()).thenReturn("Mr Harry Potter");
+        Mockito.when(appellant.getName()).thenReturn(name);
+        Mockito.when(appeal.getAppellant()).thenReturn(appellant);
+        Mockito.when(sscsCaseData.getAppeal()).thenReturn(appeal);
+        //then
+        assertEquals("Mr Harry Potter", SscsCaseDataUtils.getCaseName(sscsCaseData));
+    }
+
+    @Test
+    void shouldGetEmptyCaseNameWhenAppealNull() {
```

Comment:
Again is this the correct behaviour? Seems really strange to me.

Reply:
Address this for MVP

Reply:
Deleted.